src/lib/detection_algorithms/detectionUtils.js

```diff
@@ -0,0 +1,35 @@
+//fisherYates http://stackoverflow.com/questions/2380019/generate-unique-random-numbers-between-1-and-100 
+module.exports.generateTestIndecies = function(percent, taggedFramesCount) {
+    //range call http://stackoverflow.com/questions/3895478/does-javascript-have-a-method-like-range-to-generate-an-array-based-on-suppl
+    taggedFrameIndecies = Array.apply(null, Array(taggedFramesCount)).map(function (_, i) {return i;});
+    for (i = taggedFrameIndecies.length-1; i > 1  ; i--){
```

Comment:
Note that you don't need to go all the way down to 1, just until the portion that you need for the testIndecies